handlers/handlers_test.go

```diff
@@ -216,7 +217,7 @@ func (suite *HandlerTestSuite) TestGetRequestTokenExtractStrategy() {
 
 func (suite *HandlerTestSuite) TestListVersion() {
 
-	req, err := http.NewRequest("GET", "http://localhost:8080/v1/version", nil)
+	req, err := http.NewRequest("GET", "http://localhost:8080/v1/version?key=st", nil)
```

Comment:
here in this test you are requesting the key from the url. Are  we going to support it for ever?